tests/ui/generics/empty-generic-brackets-equiv.rs

```diff
@@ -0,0 +1,32 @@
+//! Test that empty type parameter list <> is equivalent to no type parameters
+//!
+//! Verifies that empty angle brackets <> are syntactically valid and equivalent
+//! to omitting type parameters entirely across various language constructs.
+
+//@ run-pass
+
+struct S;
+trait T {} //~ WARN trait `T` is never used
+enum E {
+    V,
+}
```

Comment:
Problem: I assume you ran rustfmt on this test file, but note that all the `<>` actually went away, which regresses test intention.